In [13]:
import pandas as pd
import numpy as np
from pathlib import Path

# Load the clean data into a dataframe and show the first few rows
spain_data_path = Path('results/spain_clean.csv')
spain_clean_df = pd.read_csv(spain_data_path)
spain_clean_df.head()


Date  Season                home          visitor score  hgoal  vgoal
0  1929-02-10    1928     Arenas de Getxo  Atletico Madrid   2-3      2      3
1  1929-02-10    1928  Espanyol Barcelona       Real Union   3-2      3      2
2  1929-02-10    1928         Real Madrid        CE Europa   5-0      5      0
3  1929-02-10    1928       Real Sociedad  Athletic Bilbao   1-1      1      1
4  1929-02-12    1928    Racing Santander     FC Barcelona   0-2      0      2

In [22]:
spain_clean_df['home']


0           Arenas de Getxo
1        Espanyol Barcelona
2               Real Madrid
3             Real Sociedad
4          Racing Santander
                ...        
26100            Granada CF
26101            CA Osasuna
26102          FC Barcelona
26103            Sevilla FC
26104         Real Sociedad
Name: home, Length: 26105, dtype: object

In [33]:
barca_home_games = spain_clean_df['home'] == 'FC Barcelona'
barca_visitor_games =  spain_clean_df['visitor'] == 'FC Barcelona' 
barca_home_games.value_counts()




False    24634
True      1471
Name: home, dtype: int64

FC Barcelona played 1,471 home games since 1929.

They should also play the same amount of visitor games. 

In [35]:
print(barca_visitor_games.value_counts())

barca_total_games = barca_home_games.value_counts() + barca_visitor_games.value_counts()
print(barca_total_games)

False    24634
True      1471
Name: visitor, dtype: int64
False    49268
True      2942
dtype: int64
